app/src/main/java/com/kanshu/kanshu/ExerciseAdapter.java

```diff
@@ -0,0 +1,83 @@
+package com.kanshu.kanshu;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.kanshu.kanshu.model.Exercise;
+
+import java.util.List;
+
+/**
+ * Created by alouanemed on 07-02-2015.
+ */
+public class ExerciseAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
+    private List<Exercise> exercisesList;
+
+    // Provide a reference to the views for each data item
+    // Complex data items may need more than one view per item, and
+    // you provide access to all the views for a data item in a view holder
+    public class UserExercisesViewHolder extends RecyclerView.ViewHolder {
+        // each data item is just a string in this case
+
+        //Exo refers to exericice
```

Comment:
I presume you mean "Ex" not "Exo".

However, I think you should just use the full word "exercise" in the variable if you have to add a comment to explain what the abbreviation means.